src/components/CategoryMonth.tsx

```diff
@@ -101,24 +84,28 @@ const CategoryMonth = ({
           </td>
           <td
             className={`whitespace-nowrap px-1 pb-2 text-sm text-right ${
-              allocation < 0 ? 'negative' : 'text-gray-500'
+              category.allocation < 0 ? 'negative' : 'text-gray-500'
```

Comment:
I think initialising the variables `{ allocation, balance, spent } = category` before the `return` and then using it here as `allocation` (and `balance` & `spent`)  like before is more legible. What do you think?

Reply:
That makes a lot of sense, implemented.